client/app/scripts/superdesk-archive/module.js

```diff
@@ -217,7 +217,7 @@ define([
          * and Killed.
          *
          * @param {Object} item
-         * @return true if the state of the item is in one of the published states, false otherwise.
+         * @return boolean if the state of the item is in one of the published states, false otherwise.
```

Comment:
The original docstring was fine as it was, although if the intent is to describe the method's return type, it should be enclosed in curly braces.

```js
/**
 * ...
 * @return {boolean} true if ..., false otherwise
 */